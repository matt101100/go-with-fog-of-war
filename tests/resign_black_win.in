place A1
place A2
place A3
resign
