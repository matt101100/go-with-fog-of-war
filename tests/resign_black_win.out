Black wins!
A1A2A3
Thank you for playing!
